api/server.go

```diff
@@ -89,6 +89,10 @@ const (
 )
 
 var (
+	// ErrInvalidAuth is returned when the supplied credentials for a protected
+	// endpoint are invalid.
+	ErrInvalidAuth = errors.New("invalid auth")
```

Comment:
`nit` this error is user facing and one they're very likely to see, so "invalid auth" might be a bit crude, maybe one of the following "bad credentials", "authentication failed", "unauthorized: bad credentials"?